src/test/run-promtool-unittests.sh

```diff
@@ -5,7 +5,11 @@ set -ex
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 : ${CEPH_ROOT:=$SCRIPTPATH/../../}
 
-sudo docker run --rm \
+
+podman_or_docker=$1
+shift
+
+sudo ${podman_or_docker} run --rm \
          -v "$CEPH_ROOT":/ceph \
          --name=promtool \
          --network=host \
```

Comment:
Do we need to specify a network?

Reply:
I wonder whether we should do a "podman/docker rmi dnanexus/promtool:2.9.2" so we don't leave an unwanted container around?

Reply:
okay, I will drop this change from this PR so this changeset can focus on the support on focal.

Reply:
You'd need to explain that comment because you give no context AFAICT.

Reply:
@badone would you kindly take another look?